docs/design/2020-09-30-index-usage-information.md

```diff
@@ -0,0 +1,159 @@
+# Proposal:
+
+- Author(s):     [rebelice](https://github.com/rebelice)
+- Last updated:  Sep. 30, 2020
+- Discussion at: N/A
+
+## Abstract
+
+This document describes the design of recording index usage information.
+
+## Background
+
+There may be unused indexes in the database. In addition, modifying database tables, indexes, and query statements may cause some indexes to not be used in the future. Users may want to clear these long-unused indexes to relieve storage and performance pressure.
+
+Related issues:
+
+- https://github.com/pingcap/tidb/issues/14998
+- https://github.com/pingcap/tidb/issues/14607
+- https://github.com/pingcap/tidb/issues/17508
+- https://asktug.com/t/topic/34614/12
+
+## Proposal
+
+### SCHEMA_INDEX_USAGE
+
+Design system tables to record index usage information. The system table is designed as follows:
+
+1. The system table is located in `mysql` database and is named `SCHEMA_INDEX_USAGE`.
+2. Columns of `SCHEMA_INDEX_USAGE`:
+
+| Column name  | Data type | Description |
+|--------------|-----------|-------------|
+| TABLE_SCHEMA | varchar   | Name of the database on which the table or view is defined.|
+| TABLE_NAME   | varchar   | Name of the table or view on which the index is defined.|
+| INDEX_NAME   | varchar   | Name of the index.|
+| QUERY_COUNT  | longlong  | Number of the SQL using this index.|
+| ROWS_SELECTED| longlong  | Number of rows read from the index. We can check the selectivity of the index through `ROWS_READ` / `QUERY_COUNT`.|
```

Comment:
I doubt that `ROWS_READ` / `QUERY_COUNT` means the selectivity of the index.
A `QUERY_COUNT` may lead many of `ROWS_READ`, `ROWS_READ` / `QUERY_COUNT` just means the average fetched rows count of each query.

Reply:
Makes sense, I will change its description.